libs/coroutines/include/hpx/coroutines/coroutine.hpp

```diff
@@ -96,6 +97,19 @@ namespace hpx { namespace threads { namespace coroutines {
             return impl_.set_thread_data(data);
         }
 
+#ifdef HPX_HAVE_LIBCDS
+        std::array<std::size_t, 3> get_libcds_data() const
```

Comment:
A typedef for `std::array<std::size_t, 3>` might be useful.

Reply:
Sure, I will fix this later today.

Reply:
Ok, now it is fixed.